webpack.config.js

```diff
@@ -27,6 +27,9 @@ const BUILD_PATH = __dirname + "/resources/frontend_client";
 const WEBPACK_BUNDLE = process.env.WEBPACK_BUNDLE || "development";
 const devMode = WEBPACK_BUNDLE !== "production";
 const useFilesystemCache = process.env.FS_CACHE === "true";
+const shouldUseEslint =
+  process.env.WEBPACK_BUNDLE !== "production" &&
+  process.env.DISABLE_ESLINT !== "true";
```

Comment:
Is there a way (without causing a hardship) to stick a with "positive" name?

Double negative [can be confusing at times](https://ariya.io/2016/04/api-names-and-begging-the-negatives).

Reply:
Good point, I renamed it to `USE_ESLINT`